src/star.css

```diff
@@ -0,0 +1,29 @@
+.starsMenu {
+  display: flex;
+  justify-content: space-between;
+  margin: 16px;
+}
```

Comment:
Blocks should not have external indents

Reply:
not fixed

Let's take an example:
https://en.bem.info/methodology/css/#external-geometry-and-positioning